src/Sushi.Mediakiwi.Data/Repositories/Sql/NotificationRepository.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Sushi.Mediakiwi.Data.MicroORM;
+using sql = Sushi.Mediakiwi.Data.Sql;   
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sushi.Mediakiwi.Data.Repositories.Sql
+{   
+    /// <summary>
+    /// Provides methods to store and retrieve instances of <see cref="Notification"/> from a Sql database.
+    /// </summary>
+    public class NotificationRepository : INotificationRepository
+    {
+        private Sushi.MicroORM.Connector<sql.Notification> connector = new Sushi.MicroORM.Connector<sql.Notification>();
+        
+        public Notification Save(Notification notification)
+        {
+            var sqlNotification = new sql.Notification(notification);
+            connector.Save(sqlNotification);
+            return sqlNotification;
+        }
+
+        public async Task<Notification> SaveAsync(Notification notification)
+        {
+            var sqlNotification = new sql.Notification(notification);
+            await connector.SaveAsync(sqlNotification);
```

Comment:
Why no ConfigureAwait here ?

Reply:
no need in .net core for configure awaits